src/runtime/internal/dev.ts

```diff
@@ -256,11 +256,47 @@ export class SvelteComponentTyped<
 	 */
 	$$slot_def: Slots;
 
-	constructor(options: IComponentOptions<Props>) {
+	constructor(options: ComponentConstructorParams<Props>) {
 		super(options);
 	}
 }
 
+/**
+ * Convenience type to get the type of a Svelte component. Useful for example in combination with
+ * dynamic components and `<svelte:element>`.
```

Comment:
Just to add some clarity between them if my current understanding is correct

```suggestion
 * dynamic components using `<svelte:component>` and elements using `<svelte:element>`.
```

Reply:
Oops I meant `<svelte:component>` only. I adjusted your suggestion.